### PR TITLE
Functional part in README

### PR DESCRIPTION
**Pull request by KyDev25**
Add functional part in README.

### CODE REVIEW

**Comment by Magali-D** [comment]
Shouldn't we add a "Get started" part with the basic command to setup the project?

File: README.md [hunk 1]
```diff
@@ -9,3 +9,117 @@
...
+## The Different Pages in our site and application
+
+### The Home Page
+
+The home page provides an overview of the categories as well as the "Highlanders of the moment" which will contain products selected in the Backoffice.
+So, this page contains all categories information.
+
+### The Category Page
+
+The category page is the main access point to the product catalogue in one category.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
The category page is the main access point to the product catalogue of a category.
```

File: README.md [hunk 2]
```diff
@@ -9,3 +9,117 @@
...
 [Tracker - Kanban board](https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2)
+
+# FUNCTIONAL PART
+
+## Introduction
+
+_ÀIRNEIS is a Scottish company and they are resellers of furniture designed by Scottish designers._
+
+We want to create our own e-commerce solution for an international clientele with products they already have locally.
+To do this, we created an e-commerce site as well an apllication, and a Backoffice to manage the content.
```
Comment (Magali-D) [inline README.md]:
```suggestion
To do this, we created an e-commerce site as well an application, and a Backoffice to manage the content.
```

File: README.md [hunk 3]
```diff
@@ -9,3 +9,117 @@
...
+So, this page contains all categories information.
+
+### The Category Page
+
+The category page is the main access point to the product catalogue in one category.
+So, this page contains information about products in a single category.
+
+### The Search Page
+
+The product page is the most important page on which the customer can purchase a product.
```
Comment (barnabepilliaudin) [inline README.md:36]: Can you reformulate
```suggestion
The search page is accessible everywhere from the site header.
So, this page allows users to research a specific product or type of product. The user can also sort the result by price or newness.
```

File: README.md [hunk 4]
```diff
@@ -9,3 +9,117 @@
...
+So, this page contains all information about a single product with the possibility to add a product in the card.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion

```

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]
Reviewed first part

File: README.md [hunk 5]
```diff
@@ -9,3 +9,122 @@
...
+So, this page contains information about products in a single category.
+
+### The Product Page
+
+The product page is the most important page on which the customer can purchase a product.
+So, this page contains all information about a single product.
+
+### The Search Page
+
+The search page is accessible everywhere from the site header.
```
Comment (barnabepilliaudin) [inline README.md:41]:
```suggestion
The search page is accessible everywhere from the site header.
```

File: README.md [hunk 6]
```diff
@@ -9,3 +9,122 @@
...
+So, this page allows them to research a specific product or type of product. The user also be able to sort the result by price or newness.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page allows them to research a specific product or type of product. The user also be able to sort the result by price or newness.
```

File: README.md [hunk 7]
```diff
@@ -9,3 +9,122 @@
...
+
+### The Shopping Cart Page
+
+The shopping cart page is accessible to all users, logged or not, they can manage their shopping cart.
+So, this page contains the list of added products and presents the total to be paid.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page contains the list of added products and presents the total to be paid.
```

File: README.md [hunk 8]
```diff
@@ -9,3 +9,122 @@
...
+
+### The Checkout Page
+
+The checkout page comes after the shopping cart page.
+So, this page contains all the information about the delivery or order.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page contains all the information about the delivery or order.
```

File: README.md [hunk 9]
```diff
@@ -9,3 +9,122 @@
...
+He contains several steps like:
+
+- Delivery.
+- Order.
+- Payment accepted.
+
+### The Registration Page
+
+The registration page allows all users to register on the site or application.
+So, this page contains mandatory information for a user before registering on the site or application.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page contains mandatory information for a user before registering on the site or application.
```

File: README.md [hunk 10]
```diff
@@ -9,3 +9,122 @@
...
+
+### The Login Page
+
+The login page allows all users who already have an account to login to the site or application.
+So, this page therefore contains mandatory information for full access to the site or application.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page therefore contains mandatory information for full access to the site or application.
```

File: README.md [hunk 11]
```diff
@@ -9,3 +9,122 @@
...
+
+### The Account Settings Page
+
+The account settings allow a user to modify their account information and profile.
+So, this page contains main and additional information of the logged in user.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page contains all the information of the logged in user.
```

File: README.md [hunk 12]
```diff
@@ -9,3 +9,122 @@
...
+
+### My Orders Page
+
+My orders page allows a user to see their past orders, grouped by year.
+So, this page contains a list of all orders with the status like:
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page contains a list of all orders with the status like:
```

File: README.md [hunk 13]
```diff
@@ -9,3 +9,122 @@
...
+
+- In progress
+- Delivered
+- Cancelled
+
+### The Contact Page
+
+The contact page is a simple page where all users can write to the site and application managers a simple contact form.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
The contact page is a simple page where all users can write to administrators with a simple contact form.
```

File: README.md [hunk 14]
```diff
@@ -9,3 +9,122 @@
...
+So, this page contains minimal information if a user has a problem with the site or a recommendation.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
This page contains minimal information if a user has a problem with the site or a recommendation.
```

File: README.md [hunk 15]
```diff
@@ -9,3 +9,122 @@
...
+
+## The Backoffice
+
+The back office allows the administrator to see all the information on a product, a category or an user.
+Based on CRUD method, we can create, read, update, or delete an item.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
An administrator can see, create, update, or delete an item.
```

File: README.md [hunk 16]
```diff
@@ -9,3 +9,122 @@
...
+
+## Miscellaneous
+
+### Menu
+
+Users have different access to the burger menu if they are logged in or not.
+
+### Design System
+
+We use a design system with a library of component graphics.
```
Comment (barnabepilliaudin) [inline README.md]:
```suggestion
We use a design system with a library of  graphics component.
```

**Review by BarnabePILLIAUDIN — CHANGES_REQUESTED** [review]
Thanks considering the following changes

**Review by BarnabePILLIAUDIN — APPROVED** [review]
は、私にはよく見えますよ (LGTM 😀)

**Review by Magali-D — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (KyDev25) [reply README.md]: Thank you!

**Review by Magali-D — APPROVED** [review]

File: README.md [hunk 17]
```diff
@@ -9,3 +9,122 @@
 # Jira link
 
 [Tracker - Kanban board](https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2)
+
+# FUNCTIONAL PART
```
Comment (cedricgautier) [inline README.md]:
```suggestion
FUNCTIONALITIES
```

File: README.md [hunk 18]
```diff
@@ -9,3 +9,122 @@
...
+
+## Introduction
+
+_ÀIRNEIS is a Scottish company and they are resellers of furniture designed by Scottish designers._
+
+We want to create our own e-commerce solution for an international clientele with products they already have locally.
+To do this, we created an e-commerce site as well an application, and a Backoffice to manage the content.
+All with a secure and maintainable payment system over time.
+
+## The Different Pages in our site and application
```
Comment (cedricgautier) [inline README.md]:
```suggestion
## What does our app contain ?
```

File: README.md [hunk 19]
```diff
@@ -9,3 +9,122 @@
 # Jira link
 
 [Tracker - Kanban board](https://kylianbarthelemy.atlassian.net/jira/software/projects/AIRNEIS/boards/2)
+
+FUNCTIONALITIES
```
Comment (cedricgautier) [inline README.md]:
```suggestion
```

**Review by cedricgautier — CHANGES_REQUESTED** [review]

**Review by cedricgautier — APPROVED** [review]

**Pull request merged**